app/services/find_or_register_user.rb

```diff
@@ -0,0 +1,27 @@
+class FindOrRegisterUser
+  attr_reader :github_id
+
+  def initialize(params)
+    @github_id = params[:github_id]
+  end
+
+  def perform
+    @_user = found_user = find_user
```

Comment:
what about just using a local var here instead of instance? seems like we don't need it outside of this method?